_posts/news/2019-11-26-announcing-scalajs-1.0.0-RC1.md

```diff
@@ -0,0 +1,541 @@
+---
+layout: post
+title: Announcing Scala.js 1.0.0-RC1
+category: news
+tags: [releases]
+permalink: /news/2019/11/26/announcing-scalajs-1.0.0-RC1/
+---
+
+
+We are thrilled to announce the release of Scala.js 1.0.0-RC1!
+
+This release candidate is intended for testing purposes by as many users as possible, and as a synchronization point with library authors so that they can start upgrading in preparation for the final release.
+If no critical issue is found until the end of January 2020, this RC will become the final release.
```

Comment:
This means we're giving 2 months to people to test RC1 before the final release. Is that appropriate?